web/vite.config.ts

```diff
@@ -93,11 +93,18 @@ function config(): Plugin {
       const remoteReearthConfig = envs.REEARTH_WEB_CONFIG_URL
         ? await (await fetch(envs.REEARTH_WEB_CONFIG_URL)).json()
         : {};
+      const reqForRemoteCesiumIonToken = envs.REEARTH_WEB_CESIUM_ION_TOKEN_URL
+        ? await fetch(envs.REEARTH_WEB_CESIUM_ION_TOKEN_URL)
+        : undefined;
+      const remoteCesiumIonToken = reqForRemoteCesiumIonToken?.ok
+        ? await reqForRemoteCesiumIonToken.text()
+        : "";
```

Comment:
```suggestion
      const remoteCesiumIonToken = envs.REEARTH_WEB_CESIUM_ION_TOKEN_URL
        ? await (await fetch(envs.REEARTH_WEB_CESIUM_ION_TOKEN_URL)).text()
        : undefined;
```

I think if you fallback to an empty string and not undefined, the ternary operator below will always be true. So either this change or just remove the ternary operator below so we will always have cesiumIonAccessToken in config.

Reply:
This part has been refacted.